### DISCUSSION TITLE
implementing generic camera to proprietary CCTV server

### DISCUSSION THREAD

**Discussion by Ivica1982** [bluenviron/mediamtx#1936] [Questions]
Hi to all. 
I've been lurking for years on githuband, and learned a lot here.
Now i'm trying to acomplish something new, and I'm stuck.

As in title, I'd like to implement generic camera with RTSP video stream to proprietary CCTV sever which is using special handler for Pelco video encoder.

I'm using mediamtx on windows machine, and i got basic functionality regarding proxying video on different IP address, TCP port and path, as long I'm using VLC player for client/reproduction.

But, when I try to use same setup with CCTV server, I'm getting refused from mediamtx.

Can someone look in Wireshark capture and try to point me in right direction?

Here is wireshark capture in situation when Pelco video encoder is used with CCTV server:
![csh3u](https://github.com/bluenviron/mediamtx/assets/136166125/7e90f0f1-2333-4fbc-ab48-3acba99f9aee)

And here is generic camera thru mediamtx with CCTV server:
![echQw](https://github.com/bluenviron/mediamtx/assets/136166125/7d643db0-9b0a-4e82-98a9-8b4ed7855377)

Thanks in advance.

**Comment by aler9** [comment]
Hello, this seems like a bug, open an issue and follow instructions in order to provide data needed to replicate the bug. In my opinion, server logs with option `logLevel: debug` will be enough to identify the error and fix the server.

[a comment by an automated account is not shown]
